store/mongo/datastore_mongo.go

```diff
@@ -743,6 +743,17 @@ func (db *DataStoreMongo) GetFiltersAttributes(
 	const DbCount = "count"
 
 	cur, err := collDevs.Aggregate(ctx, []bson.M{
+		// Sample 1000 devices to get a representative sample
+		{
+			"$limit": 10000,
```

Comment:
could we make it into a const? (maybe one day we would like to make it configurable)

Reply:
Done